_blog-src/_posts/2015-08-03-policy-endpoint-security.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "apiman Policy and Endpoint Security"
+date:   2015-09-03 11:00:00
```

Comment:
Argh! Anyone in QE should not be making all these typo's   ;-)

On Mon, Aug 3, 2015 at 10:49 AM, Eric Wittmann <notifications@github.com>
wrote:

> In _blog-src/_posts/2015-08-03-policy-endpoint-security.markdown
> <https://github.com/apiman/apiman.github.io/pull/26#discussion_r36092193>:
>
> > @@ -0,0 +1,72 @@
> > +---
> > +layout: post
> > +title:  "apiman Policy and Endpoint Security"
> > +date:   2015-09-03 11:00:00
>
> Wrong date - should be 2015-08-03! :) I'll merge and fix it. No worries.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiman/apiman.github.io/pull/26/files#r36092193>.
>



-- 
Len DiMaggio (ldimaggi@redhat.com)
JBoss by Red Hat
314 Littleton Road
Westford, MA 01886  USA
tel:  978.392.3179
cell: 781.472.9912
http://www.redhat.com
http://community.jboss.org/people/ldimaggio